src/core/config/names.h

```diff
@@ -41,4 +41,6 @@ constexpr auto kCfdColumnsNumber = "columns_number";
 constexpr auto kCfdTuplesNumber = "tuples_number";
 constexpr auto kCfdMaximumLhs = "cfd_max_lhs";
 constexpr auto kCfdSubstrategy = "cfd_substrategy";
+// constexpr auto kGraphPath = "graph_path";
+constexpr auto kGFDData = "gfd";
```

Comment:
Yes, there is `./Desbordante_run --algo=gfdvalid --csv_path="input_data/graph_data/quadrangle.dot" --gfd="input_data/graph_data/quadrangle_gfd.dot"`
I had to interpret `csv_path` as the path to the graph, since this parameter is required for all algorithms